src/Symfony/Component/Cache/Adapter/AbstractAdapter.php

```diff
@@ -372,6 +373,40 @@ public function commit()
         return $ok;
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * This implementation is not atomic unless the "doIncrement()" method provided by the concrete adapter is.
+     */
+    public function increment($key, $step = 1)
+    {
+        if (!is_numeric($step)) {
+            return false;
+        }
+        $id = $this->getId($key);
+
+        try {
+            if (is_numeric($result = $this->doIncrement($id, (int) $step))) {
+                return $result;
+            }
+            CacheItem::log($this->logger, 'Failed to increment key "{key}"', array('key' => $key));
```

Comment:
Since you call this code also from decrement, this message may be misleading in some cases